Sources/SwiftInspectorVisitors/Tests/PropertyInfoSpec.swift

```diff
@@ -2,3 +2,131 @@
 // Copyright © 2021 Airbnb Inc. All rights reserved.
 
 import Foundation
+import Nimble
+import Quick
+import SwiftInspectorTestHelpers
+import SwiftSyntax
+
+@testable import SwiftInspectorVisitors
+
+final class PropertyInfoParadigmSpec: QuickSpec {
+
+  let undefinedConstantTestCase = PropertyInfo.Paradigm.undefinedConstant
+  let undefinedConstantTestCaseData = """
+    {
+      "caseValue": 0
```

Comment:
By encoding the cases as numbers rather than strings we make the encoded data smaller and reduce computation.